app/src/main/java/org/imozerov/streetartview/ui/catalog/ArtListFragment.kt

```diff
@@ -0,0 +1,82 @@
+package org.imozerov.streetartview.ui.catalog
+
+import android.os.Bundle
+import android.support.v4.app.Fragment
+import android.support.v7.widget.LinearLayoutManager
+import android.util.Log
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlinx.android.synthetic.main.fragment_art_list.view.*
+import org.imozerov.streetartview.R
+import org.imozerov.streetartview.StreetArtViewApp
+import org.imozerov.streetartview.storage.DataSource
+import org.imozerov.streetartview.ui.interfaces.Filterable
+import org.imozerov.streetartview.ui.model.ArtObjectUi
+import rx.Subscription
+import rx.android.schedulers.AndroidSchedulers
+import javax.inject.Inject
+
+class ArtListFragment : Fragment(), Filterable {
+    private val TAG = "AlertListFragment"
```

Comment:
logt in android studio generates tag with string literal. I prefer to stick to this strategy. I'll fix the typo.